.vscode/launch.json

```diff
@@ -6,7 +6,7 @@
             "type": "python",
             "request": "launch",
             "stopOnEntry": true,
-            "pythonPath": "${config:python.pythonPath}",
+            "python": "${command:python.interpreterPath}",
```

Comment:
Do these changes mean user must be in virtual environment?

Reply:
It will depend on the current environment configuration, which includes whether or not you are using a virtual environment. Thanks for the review. Looks like  "python": "${command:python.interpreterPath}", newer and was introduced more recently. This looks more useful when a user wants to point to a virtual environment